### DISCUSSION TITLE
Pyrin low hashrate on CMP cards (50hx , 90hx)

### DISCUSSION THREAD

**Discussion by albertoxa** [Lolliedieb/lolMiner-releases#2010] [General]
hello , could you please consider fixing the low hashrate problem with cmp cards on the pyrin algo ? .

**Comment by Maythecoffee** [comment]
and 2080ti can mine perfectly but somerthing wrong with cmp50

**Comment by XRegem** [comment]
I join the question. Why is the speed on the CMP 50 so low?After all, this is essentially a 2080ti?
